# Observational Studies
<b>2x2 Contingency Table</b>
|              | Disease<br> (or Outcome) | Disease-free |
|--------------|---------|--------------|
| Exposed      | a       | c            |
| Unexposed    | b       | d            |

## Odds Ratio using Case-Control Incidence

This is the relative chance of developing the disease by being exposed to a risk factor

Calculation:<br>
Disease Odds in the Exposed Group $\frac{a}{c}$ divided by Disease Odds in the Unexposed Group $\frac{b}{d}$

$\frac{a/c}{b/d}$ or $\frac{a*d}{b*c}$ utilising the "Cross Product Ratio" for eliminating fractions<br>
Retrospective observational studies produce OR with no predictive power

In [1]:
import os
os.chdir("/Applications/Stata/utilities")
from pystata import config
config.init("se")


  ___  ____  ____  ____  ____ ®
 /__    /   ____/   /   ____/      StataNow 18.5
___/   /   /___/   /   /___/       SE—Standard Edition

 Statistics and Data Science       Copyright 1985-2023 StataCorp LLC
                                   StataCorp
                                   4905 Lakeway Drive
                                   College Station, Texas 77845 USA
                                   800-782-8272        https://www.stata.com
                                   979-696-4600        service@stata.com

Stata license: Unlimited-user network, expiring  9 Sep 2025
Serial number: 501809305305
  Licensed to: Mujie
               

Notes:
      1. Unicode is supported; see help unicode_advice.
      2. Maximum number of variables is set to 5,000 but can be increased;
          see help set_maxvar.


Suppose
|                 | Stroke  | No Stroke |
|-----------------|---------|--------------|
| With Aura       | 23      | 27           |
| Without Aura    | 51      | 69           |

In [2]:
%%stata

cci 23 51 27 69


. 
. cci 23 51 27 69
                                                         Proportion
                 |   Exposed   Unexposed  |      Total      exposed
-----------------+------------------------+------------------------
           Cases |        23          51  |         74       0.3108
        Controls |        27          69  |         96       0.2812
-----------------+------------------------+------------------------
           Total |        50         120  |        170       0.2941
                 |                        |
                 |      Point estimate    |    [95% conf. interval]
                 |------------------------+------------------------
      Odds ratio |         1.152505       |    .5609558    2.356181 (exact)
 Attr. frac. ex. |         .1323251       |   -.7826715    .5755843 (exact)
 Attr. frac. pop |         .0411281       |
                 +-------------------------------------------------
                               chi2(1) =     0.18  Pr>chi2

Key outputs are OR and 95% CI:<br>
An OR of 1.15 indicates that individuals with aura are 15% more likely to experience a stroke than individuals without aura<br>
However, the 95% CI ranges from 0.56 to 2.35, which includes 1. We are 95% confident that the true OR falls within this range, and the long-run average OR falls within the CI 95% of the time. Therefore, there is no statistical difference in OR of getting a stroke between those with aura and those without

## Risk Ratio using Case Sample Incidence
This is the relative risk of developing a disease by being exposed to a risk factor

Calculation:<br>Disease Risk in the Exposed Group $\frac{a}{a+c}$ divided by Disease Risk in the Unexposed Group $\frac{b}{b+d}$

$\frac{\frac{a}{a+c}}{\frac{b}{b+d}}$

Prospective observational studies produce RR with predictive power

Suppose
|                 | Dementia  | No Dementia |
|-----------------|---------|--------------|
| Stroke       | 18      | 15           |
| No Stroke    | 45      | 140           |

In [3]:
%%stata

csi 18 15 45 140


. 
. csi 18 15 45 140

                 |   Exposed   Unexposed  |      Total
-----------------+------------------------+-----------
           Cases |        18          15  |         33
        Noncases |        45         140  |        185
-----------------+------------------------+-----------
           Total |        63         155  |        218
                 |                        |
            Risk |  .2857143    .0967742  |   .1513761
                 |                        |
                 |      Point estimate    |    [95% conf. interval]
                 |------------------------+------------------------
 Risk difference |         .1889401       |     .068067    .3098131 
      Risk ratio |         2.952381       |    1.589047    5.485397 
 Attr. frac. ex. |         .6612903       |    .3706919    .8176978 
 Attr. frac. pop |         .3607038       |
                 +-------------------------------------------------
                               chi2(1) =    12.4

Key outputs are RR and 95% CI:<br>
An RR of 2.95 indicates that individuals with stroke are 195% more likely to develop dementia than individuals without stroke<br>
The 95% CI ranges from 1.59 to 5.49, which excludes 1. We are 95% confident that the true OR falls within this range; the long-run average falls within the CI 95% of the time. Therefore, the difference is statistically significant; we can reject the $H_0$ that there is no difference in risk of developing dementia between those with stroke and those without.

Not only is this result statistically significant, this risk factor is also <u>clinically significant</u> since those with stroke are 3 times more likely to develop dementia

> If the disease prevalence is low i.e. disease is rare, the OR would be similar to the RR

## Meta-Analysis: See R (for Meta-Analysis)